app/modules/asset_groups/models.py

```diff
@@ -589,10 +589,13 @@ def get_pipeline_status(self):
             'summary': {},
         }
         status['summary']['complete'] = (
-            status['preparation']['complete']
-            and status['detection']['complete']
-            and status['curation']['complete']
-            and status['identification']['complete']
+            (status['preparation']['complete'] or status['preparation']['skipped'])
+            and (status['detection']['complete'] or status['detection']['skipped'])
+            and (status['curation']['complete'] or status['curation']['skipped'])
+            and (
```

Comment:
I fail to see how looking at the identification status of an AGS makes sense. An AGS never does identiification so it will always be not run

Reply:
FE just wanted consistency in the content regardless of whether each section necessarily was logical there.